packages/data-stores/src/plans/selectors.ts

```diff
@@ -14,7 +15,6 @@ import {
 	STORE_KEY,
 	FREE_PLAN_PRODUCT_ID,
 } from './constants';
-import deprecate from '@wordpress/deprecated';
```

Comment:
Moved to external dependencies (see above)

Reply:
Nitpick mode: Scope creep?

Reply:
Reverted inhttps://github.com/Automattic/wp-calypso/pull/50492/commits/6a44b523402b69d8986278e0eb065797e2486481, and opened #50666 to include these changes in a separate PR.